Makefile

```diff
@@ -89,10 +75,6 @@ mock:
 	go generate -run mockgen ./...
 
 # Deletes all executables matching the directory names in cmd/
-.PHONY: clean-go
-clean-go:
-	rm -f bin/
-
 .PHONY: clean
-clean: clean-go
-	rm -f $(DLEQ_LIB)
+clean:
+	rm -r bin/
```

Comment:
You may be able to adjust your editor here.  Mine is configured to ensure that files always have a trailing newline (newline at the end of the last, non-empty line).